### DISCUSSION TITLE
Copy data from a crashed minio cluster to a new cluster(Different specifications)?

### DISCUSSION THREAD

**Discussion by Clara12062** [minio/minio#20336] [Q&A] [answered]
Hi~

I have a Minio cluster (referred to as Minio-A), with the following format.json:
```json
{
  "version": "1",
  "format": "xl",
  "id": "bc705a4e-4f88-4f1d-8c33-db85e3abaab8",
  "xl": {
    "version": "3",
    "this": "6a667406-fbe6-4ff6-8057-384a2e1f4806",
    "sets": [
      [
        "6a667406-fbe6-4ff6-8057-384a2e1f4806",
        "0fdd4535-bf90-4859-99f9-01b6ddd1c5cd"
      ]
    ],
    "distributionAlgo": "SIPMOD+PARITY"
  }
}
```

Subsequently, I simulate the failure of Minio-A, delete it, and ensure that the data is fully preserved. Then, I deploy a new Minio cluster (Minio-B), with the following format.json:
```
{
  "version": "1",
  "format": "xl",
  "id": "49c6737d-3bb8-4caf-a001-23c729333bca",
  "xl": {
    "version": "3",
    "this": "cd124e81-bb55-4feb-a811-c82110a5ba52",
    "sets": [
      [
        "3e941dfc-3077-4148-b687-92a8ca8db424",
        "cd124e81-bb55-4feb-a811-c82110a5ba52",
        "2d7116a3-8af3-4ef4-9a51-b66c94b17a4b"
      ]
    ],
    "distributionAlgo": "SIPMOD+PARITY"
  }
}
```

Now, when I use `mc admin stat ALIAS/bucket` from minio-B, I can see the previous bucket, but I cannot see the original objects.

Is there a feasible approach to copy the data from Minio-A's disks to Minio-B in order for the old objects to be visible on Minio-B?

**Comment by klauspost** [comment, marked as answer]
> Is there a feasible approach to copy the data from Minio-A's disks to Minio-B

No. You should never operate directly on disks. It will not work. Use API for copying.

**Reply by Clara12062** [reply]
ok~ Thanks for your reply
